src/main/java/io/stargate/sgv2/jsonapi/service/processor/MeteredCommandProcessor.java

```diff
@@ -285,7 +285,9 @@ public MeterFilter enableHistogram() {
       public DistributionStatisticConfig configure(
           Meter.Id id, DistributionStatisticConfig config) {
         if (id.getName().startsWith(jsonApiMetricsConfig.metricsName())
-            || id.getName().startsWith(HISTOGRAM_METRICS_NAME)) {
+            || id.getName().startsWith(HISTOGRAM_METRICS_NAME)
+            || id.getName()
+                .startsWith(jsonApiMetricsConfig.jsonSerializationPerformanceMetricsName())) {
```

Comment:
Should this only apply to timing? Metrics?

Reply:
Yeah... Since we also have size metrics, should I enable the histogram for it?